### PR TITLE
Incremental housekeeping

### PR DESCRIPTION
**Pull request by yazug**
Number of small cleanups

- linting cleanup sort imports
- normalize doc text
- switch a number of type() == to isinstance() checks
- extend test_remove_builds for isinstance change assumption

### CODE REVIEW

File: errata_tool/erratum.py [hunk 1]
```diff
@@ -722,13 +722,14 @@ def setFileInfo(self, file_info):
         self._processResponse(r)
 
     def removeBuilds(self, buildlist):
-        if type(buildlist) is not str and type(buildlist) is not list:
```
Comment (yazug) [inline errata_tool/erratum.py]: This is the only non-trivial type()  to isinstance() conversion

from tests added in test_remove_builds.py I verified that isinstance(foo, (<type1>, <type2>)) does a is instance either type1 or type2, so this should be the clean refactor.

File: errata_tool/erratum.py [hunk 2]
```diff
@@ -722,13 +722,14 @@ def setFileInfo(self, file_info):
...
+        """Remove build(s) from advisory"""
+        if not isinstance(buildlist, (str, list)):
             raise IndexError
 
         # Removing builds
         # REFERENCE
 
-        if isinstance(buildlist, six.string_types):
```
Comment (yazug) [inline errata_tool/erratum.py]: hmm not intended to remove the six.string_types from this one, as that was here before and dealt with unicode strings in py2.

File: errata_tool/tests/test_remove_builds.py [hunk 3]
```diff
@@ -65,3 +73,18 @@ def test_builds_empty_tuple(self, monkeypatch, mock_post, advisory):
         with pytest.raises(IndexError):
             advisory.removeBuilds(())
         assert advisory._buildschanged is False
+
+    def test_asumptions(self):
+        assert isinstance('', (str, list))
```
[the remark this answers is not in the document]
Comment (yazug) [reply errata_tool/tests/test_remove_builds.py]: I can see that. it was more to verify the assumption and what all counted as list or string was correct.  I'll drop it from the patch

File: errata_tool/erratum.py (hunk 2 above)
Comment (yazug) [reply errata_tool/erratum.py]: it was my code change and I need to revert this change.

Comment (yazug) [reply errata_tool/erratum.py]: done

File: errata_tool/tests/test_remove_builds.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (yazug) [reply errata_tool/tests/test_remove_builds.py]: done

**Comment by ktdreyer** [comment]
I've pushed `1.21.0` to https://pypi.org/project/errata-tool/ with this change.

**Pull request merged**